ush/config_defaults.sh

```diff
@@ -584,7 +584,7 @@ ARCHIVE_CYCLEDEF="00 01 01 01 2100 *"
 diag_radardbz=.true.
 write_diag_2=.false.
 bkgerr_vs=1.0
-bkgerr_hzscl=0.373,0.746,1.5   #no trailing ,
+bkgerr_hzscl=0.7,1.4,2.80   #no trailing ,
```

Comment:
This is to match the new BEC, right?

Reply:
Yes, from EMC.

Reply:
Okay.  Does this set of number work for regional only? Global uses the same or another set of number? 
Do we know the equivalent horizontal length scale in km for this set of number? We'd like to know this before reducing them for RTMA.